src/consent.js

```diff
@@ -1,6 +1,3 @@
-import {
```

Comment:
`import type {CONSENT_POLICY_STATE} from '...'`?

Reply:
Is `import type` valid JS?

Reply:
No. I think @jridgewell may have missed that all these are still JS files 😬